cardinal/ecs/world.go

```diff
@@ -458,6 +465,8 @@ func (w *World) AddTransaction(id transaction.TypeID, v any, sig *sign.SignedPay
 // Tick performs one game tick. This consists of taking a snapshot of all pending transactions, then calling
 // each System in turn with the snapshot of transactions.
 func (w *World) Tick(ctx context.Context) error {
+	tickAsString := strconv.FormatUint(w.tick, 10)
+	w.logger.Info().Str("tick", tickAsString).Msg("Tick started")
```

Comment:
Might be slowish here due to conversion to string. zerolog doesn't support uint64, but I can convert to int.

Reply:
could also just do `fmt.Sprintf("%d", w.tick)`

Reply:
not sure if one is faster than the other though